package.json

```diff
@@ -29,44 +30,31 @@
   ],
   "license": "MIT",
   "files": [
-    "server",
     "cli",
-    "dist",
+    "src",
```

Comment:
Because vite builds the tailwind config into the output instead of getting it at runtime from a separate server, I think we need to publish the source.